doc/Makefile

```diff
@@ -7,18 +7,21 @@ SPHINXBUILD  ?= sphinx-build
 PAPER         =
 BUILDDIR      = _build
 
-# Run sequential by default, unless SPHINX_NUMJOBS is set.
-SPHINX_NUMJOBS ?= 1
-
 ifneq ($(EXAMPLES_PATTERN),)
     EXAMPLES_PATTERN_OPTS := -D sphinx_gallery_conf.filename_pattern="$(EXAMPLES_PATTERN)"
 endif
 
+ifeq ($(CI), true)
+    SPHINX_NUMJOBS_NOPLOT_DEFAULT=2
+else
+    SPHINX_NUMJOBS_NOPLOT_DEFAULT=auto
+endif
```

Comment:
I'm still getting stalling issues with MacOS using `-jauto`. May we have a `($(shell uname), Darwin)` condition?

```suggestion
else ($(shell uname), Darwin)
    SPHINX_NUMJOBS_NOPLOT_DEFAULT=1
else
    SPHINX_NUMJOBS_NOPLOT_DEFAULT=auto
endif
```